config/adventurebackpack.cfg

```diff
@@ -46,8 +46,8 @@ graphics {
     # Enable rendering for tools in the backpack tool slots [default: true]
     B:"Enable Tools Render"=true
 
-    # Show hovering text on fluid tanks? [default: false]
-    B:"Hovering Text"=false
+    # Show hovering text on fluid tanks? [default: true]
```

Comment:
Was the default actually changed in the adv backpack code? Ie if you delete the config does it default tru?

Reply:
I did enable it in the mod assuming it was disabled due to being a broken feature. AFAIK when the code runs if no config exists, it's set to the default.